pkgs/applications/office/watson/default.nix

```diff
@@ -13,6 +13,15 @@ buildPythonApplication rec {
     sha256 = "sha256-/AASYeMkt18KPJljAjNPRYOpg/T5xuM10LJq4LrFD0g=";
   };
 
+  patches = [
+    # https://github.com/TailorDev/Watson/pull/473
+    (pkgs.fetchpatch {
```

Comment:
```suggestion
    (fetchpatch {
```

Reply:
That gives me the following error message:

```
> nix-build -A watson
error: undefined variable 'fetchpatch'

       at /home/user/path/nixpkgs/pkgs/applications/office/watson/default.nix:18:6:

           17|     # https://github.com/TailorDev/Watson/pull/473
           18|     (fetchpatch {
             |      ^
           19|       name = "fix-completion.patch";
(use '--show-trace' to show detailed location information)
```

Reply:
You also need to add it to inputs which is basic nix knowledge. You should probably familiarize yourself more with the language.

Reply:
Perhaps it would be more useful to link the proper section of the documentation instead of using harsh words.

@voidus: Would you please try adding `fetchpatch` next to `fetchFromGitHub` in the header?